ynr/settings/testing.py

```diff
@@ -29,7 +30,7 @@ def __getitem__(self, item):
 RUNNING_TESTS = True
 
 SECRET_KEY = "just here for testing"
-ALLOWED_HOSTS = ["candidates.democracyclub.org.uk"]
+ALLOWED_HOSTS = ["candidates.democracyclub.org.uk","*"]
```

Comment:
What did we need to add this for?

Reply:
I found that the setting as originally configured prevented the site from being accessible at `localhost:8080` or similar, as the Host header (or perhaps lack of one) was rejected. I'm unsure if the check overlaps with a `DEBUG`   setting that's more lenient, maybe, but at the point I needed to see the site on localhost, this change was needed.

Reply:
What I am getting at here is: Did we do this because we needed it to run unit tests inside the container, or did we do it because we were trying to use `ynr.settings.testing` for interactive usage (i.e: things other than running unit tests?)

If the first one, then we need it.
If the second one, then we can revert this.

Reply:
Resolved by https://github.com/DemocracyClub/yournextrepresentative/commit/fb11f14cd55a5b43f86fb08ce5bd05d17afa05a7 (not yet pushed to this branch).

Reply:
Resolved by https://github.com/DemocracyClub/yournextrepresentative/pull/2485/commits/196d5f3ffca6f529a661ca5a72b23b04c04b2ee6. This change has been reverted. The container now uses `ynr.settings` for interactive invocations, which were the prompt for this earlier change.